modules/module/server-ts/ServerModule.ts

```diff
@@ -5,6 +5,7 @@ import { Express } from 'express';
 import { ConnectionParamsOptions } from 'subscriptions-transport-ws';
 import { IResolvers } from 'graphql-tools';
 import CommonModule, { CommonModuleShape } from '@gqlapp/module-common';
+import { GraphQLSchema } from 'graphql';
```

Comment:
All the GraphQL-related code/logic/exports should be located in the newly created `graphql` module.

Reply:
Done